microsoft-edge/devtools-guide-chromium/crash-analyzer/index.md

```diff
@@ -6,108 +6,108 @@ ms.author: msedgedevrel
 
+<!-- TODO: add a diagram explaining how the system works -->
 
-<!-- ====================================================================== -->
-## Overview of steps to produce and analyze stack traces
-
-To analyze crash stack traces in the **Crash analyzer** tool (end-to-end high-level steps):
-
-1. It's recommended that you store source maps using Azure Artifacts Symbol Server.  See [Store source maps using Azure Artifacts Symbol Server](#store-source-maps-using-azure-artifacts-symbol-server), below.
 
-1. Use the [Crash Analyzer Support](https://www.npmjs.com/package/@microsoft/edge-devtools-crash-analyzer-support) npm package in your web app.  The Crash Analyzer Support package produces stack traces in the format that's needed by the **Crash analyzer** tool.  To use the library that's in the Crash Analyzer Support npm package, you import the Crash Analyzer Support library, and then call `installErrorStackModuleAnnotations(Error);`.  See [Use the Crash Analyzer Support npm package to collect stack traces](#use-the-crash-analyzer-support-npm-package-to-collect-stack-traces), below.
-
-1. Put in place some kind of production stack trace collection system, such as Application Insights.  This way, when your web app crashes on a customer's machine, the stack trace that was thrown when the crash occurred is recorded somewhere in your system.  See [Use Application Insights to receive a stack trace from the web browser](#use-application-insights-to-receive-a-stack-trace-from-the-web-browser), below.
-
-1. Then, later, use the **Crash analyzer** tool to unminify the specially formatted stack traces that you collected in production, which are specially formatted.  This step makes use of the Azure Artifacts Symbol Server configuration to unminify the stack trace.  This enables you to see exactly where the crash happened in the code, as if you were using the customer's machine when the crash occurred.  See [Analyze crash stack traces in the Crash analyzer tool](#analyze-crash-stack-traces-in-the-crash-analyzer-tool), below.
+<!-- ====================================================================== -->
+## Collect stack traces in production
 
+The **Crash analyzer** tool doesn't collect stack traces for you. You must first collect stack traces from your web app by using the tools and services that are available to you.  Here are some ways to collect stack traces in production:
 
-<!-- ====================================================================== -->
-## Store source maps using Azure Artifacts Symbol Server
+* We recommend using a telemetry system such as [Azure Monitor Application Insights](/azure/azure-monitor/app/app-insights-overview) which can capture unhandled error information from your JavaScript code in production.
```

Comment:
this "we" might be ok per
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/w/we
but still, per that page, will try to rewrite to avoid.